packages/webcomponents/src/components/checkout/checkout-core.tsx

```diff
@@ -6,11 +6,11 @@ import { Checkout, ICheckout, ICheckoutCompleteResponse, ILoadedEventResponse }
 import { ComponentError, ComponentErrorCodes, ComponentErrorSeverity } from '../../api/ComponentError';
 import { insuranceValues, insuranceValuesOn, validateInsuranceValues } from '../insurance/insurance-state';
 import { BillingFormFields } from '../billing-form/billing-form-schema';
-import { Button, StyledHost } from '../../ui-components';
+import { Button, StyledHost, Skeleton } from '../../ui-components';
 
 @Component({
   tag: 'justifi-checkout-core',
-  styleUrls: ['checkout-core.css', 'header.css'],
+  styleUrls: ['checkout-core.css', 'header.css', '../../styles/animations.css'],
```

Comment:
Instead maybe we can put the animation into the `modified-bootstrap`/`StyledHost`?